ops/charm.py

```diff
@@ -297,6 +297,20 @@ def restore(self, snapshot):
             self.unit = None
 
 
+class RelationCreatedEvent(RelationEvent):
+    """Represents the `relation-created` hook from Juju.
+
+    It's triggered by `CharmBase.on[RELATION].relation_created`.
```

Comment:
Can we please get rid of all of these notes?  It doesn't make much sense to be reverse-defining where the type is used. The documentation of where it's used should be clear about that instead.

Reply:
Agreed, I'll get rid of all of them.

Reply:
https://github.com/canonical/operator/pull/218/commits/6e5916633b97995ca88a080597dff7c6e9ceff4c